app/views/layouts/partners/navigation/_navbar.html.erb

```diff
@@ -6,7 +6,7 @@
 
   <div class="dropdown-menu dropdown-menu-lg dropdown-menu-right">
     <%= link_to '', class:"dropdown-item" do %>
-    <i class="fa fa-cog text-aqua"></i> Organization Settings
+    <i class="fa fa-cog text-aqua mr-2"></i> Organization Settings
```

Comment:
Thank you for digging into this! This has been bugging me for some time but never got around to doing anything about it.

I noticed that the spacing is still a little uneven between the elements. Do you think you can treat the text as another div element and use flex to justify them to the start, and maybe add `space-x-2` at the parent to add spacing between the icon and the text?